src/components/CreateAdministration.vue

```diff
@@ -381,6 +392,8 @@ const rules = {
   dateStarted: { required },
   dateClosed: { required },
   sequential: { required },
+  consent: { required },
+  assent: { required },
```

Comment:
Good point, let's use `requiredIf` instead of `required` (https://vuelidate-next.netlify.app/validators.html#requiredif) and require it if it's not the LEVANTE instance.